docs/pages/setup/reference/fluentd.mdx

```diff
@@ -0,0 +1,103 @@
+| `types`               | Comma-separated list of event types to forward      | `FDFWD_TYPES`               |
+| `skip-session-types`  | Comma-separated list of session event types to skip | `FDFWD_SKIP_SESSION_TYPES`  |
+| `start-time`          | Minimum event time (RFC3339 format)                 | `FDFWD_START_TIME`          |
+| `timeout`             | Polling timeout                                     | `FDFWD_TIMEOUT`             |
+| `cursor`              | Start cursor value                                  | `FDFWD_CURSOR`              |
+| `debug`               | Debug logging                                       | `FDFWD_DEBUG`               |
+| `config`              | Path to TOML configuration file                     | `FDFWD_CONFIG`              |
+
+TOML configuration key names are the same as CLI args. 
+
+Teleport and Fluentd variables can be grouped into sections in a TOML file:
+
+```
+[teleport]
+addr = "localhost:3025"    # Equals to teleport-addr root key
+identity = "tmp/identity"  # Equals to teleport-identity root key
+```
+
+## Special arguments
+
+There are two special arguments which might be used on the initial setup:
+
+- `--dry-run` argument forces `teleport-event-handler` to simulate event export. It will read all the events, but won't send them to `fluentd.`
+- `--exit-on-last-event` argument forces `teleport-event-handler` to exit when last event gots processed. This might be useful for initial export.
+
+## URLs
+
+`teleport-event-handler` sends two types of events: normal cluste-wide audit log entries and session events. 
+You need to specify two URLs: `fluentd-url` is used for base Audit Log entries. `fluentd-session-url` is 
+required to send session events. Full session url would be: `%fluentd-session-url%.%session id%.log`.
+
+There should be two `match` entries in the fluentd configuration:
+
+```
+# This matches Audit Log events
+<match test.log>
+  @type stdout
+</match>
+
+# This matches Session events
+<match session.*.log> 
+  @type stdout
+</match>
+```
+
+## Do not forget to set time range
+
+By default, all events starting from the current moment will be exported. If you want to export an older events, you have to pass `--start-time` argument.
+
+This will start an export from May 5 2021 (RFC3339 time):
+
+```sh
+teleport-event-handler start --config teleport-event-handler.toml --start-time "2021-05-05T00:00:00Z"
+```
+
+This will export new events from a moment the service did start:
+
+```sh
+teleport-event-handler start --config teleport-event-handler.toml
+```
+
+Note that start time can be set only once, on the first run. If you want to change the start time frame later, remove plugin state dir which you had specified in `storage-dir` argument.s
+
+## Export print events
+
+Session print events are skipped by default. If you still want to export them, specify `--skip-session-types=''` argument.
```

Comment:
We should probably skip `DesktopRecording` events, too. They're the graphical equivalent of `SessionPrint` events.

Reply:
@zmb3 Okay, I'll create the separate PR for that.